### PR TITLE
Replacing cpu timing calls from gl to std::chrono

### PR DESCRIPTION
**Pull request by samcake**
The method to measure CPU batch time from a gpu::query has been changed to std::chrono instead of from using gl (glGetInteger64v(GL_TIMESTAMP, ...).
This is two less opengl calls which were actually showing up in profiles.

## TEST PLAN 
This is simply replacing one method by another to measure the Batch time so if the batch time is visible in stats as normal  then this means that it s a pass.
this should be verified on mac and pc platforms
and check that andoird build works correctly too

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12862/HighFidelity-Beta-PR13972-840c8c9.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12862/HighFidelity-Beta-PR13972-840c8c9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/12862/HighFidelity-Beta-Interface-PR13972-840c8c9.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/12871/HighFidelity-Beta-PR13972-7758693.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12871/HighFidelity-Beta-PR13972-7758693.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/12871/HighFidelity-Beta-Interface-PR13972-7758693.dmg

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
CPU times are working and visible.

File: libraries/gpu-gl-common/src/gpu/gl/GLBackendQuery.cpp [hunk 1]
```diff
@@ -25,61 +25,61 @@ static bool timeElapsed = false;
...
         } else {
             glQueryCounter(glquery->_endqo, GL_TIMESTAMP);
         }
+#endif
 
         --_queryStage._rangeQueryDepth;
-        GLint64 now;
-        glGetInteger64v(GL_TIMESTAMP, &now);
-        glquery->_batchElapsedTime = now - glquery->_batchElapsedTime;
+        auto duration_ns = (std::chrono::high_resolution_clock::now() - glquery->_batchElapsedTimeBegin);
```
Comment (jherico) [inline libraries/gpu-gl-common/src/gpu/gl/GLBackendQuery.cpp]: if you want this to be guaranteed to be nanoseconds you need to do `... = std::chrono::duration_cast<std::chrono::nanoseconds>(std::chrono::high_resolution_clock::now() - glquery->_batchElapsedTimeBegin)`

**Pull request merged**